src/v2/util/BaseLoginRouter.js

```diff
@@ -120,44 +113,22 @@ function (Okta, BrowserFeatures, Settings,
       });
 
       this.listenTo(this.appState, 'change:remediationFailure', function (appState, err) {
-        RouterUtil.routeAfterAuthStatusChangeError(this, err);
-      });
-
-      this.listenTo(this.appState, 'change:remediationSuccess', function (appState, trans) {
```

Comment:
- maybe we need `change:remediationSuccess` for the resp changes when moving through the pipeline?
- also trivial handler for `change:introspectSuccess` to copy `introspectSuccess` into `remediationSuccess`. Doesn't seem ideal but we could add comments to explain the reason.